wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/Form.kt

```diff
@@ -0,0 +1,765 @@
+import kotlin.Deprecated
+import kotlin.DeprecationLevel
+import kotlin.Int
+import kotlin.jvm.JvmField
+import okio.ByteString
+
+data class Form(
+  @field:WireField(
+    tag = 1,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}ButtonElement#ADAPTER"
+  )
+  val button_element: ButtonElement? = null,
+  @field:WireField(
+    tag = 2,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}LocalImageElement#ADAPTER"
+  )
+  val local_image_element: LocalImageElement? = null,
+  @field:WireField(
+    tag = 3,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}RemoteImageElement#ADAPTER"
+  )
+  val remote_image_element: RemoteImageElement? = null,
+  @field:WireField(
+    tag = 4,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}MoneyElement#ADAPTER"
+  )
+  val money_element: MoneyElement? = null,
+  @field:WireField(
+    tag = 5,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}SpacerElement#ADAPTER"
+  )
+  val spacer_element: SpacerElement? = null,
+  @field:WireField(
+    tag = 6,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}TextElement#ADAPTER"
+  )
+  val text_element: TextElement? = null,
+  @field:WireField(
+    tag = 7,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}CustomizedCardElement#ADAPTER"
+  )
+  val customized_card_element: CustomizedCardElement? = null,
+  @field:WireField(
+    tag = 8,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}AddressElement#ADAPTER"
+  )
+  val address_element: AddressElement? = null,
+  @field:WireField(
+    tag = 9,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}TextInputElement#ADAPTER"
+  )
+  val text_input_element: TextInputElement? = null,
+  @field:WireField(
+    tag = 10,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}OptionPickerElement#ADAPTER"
+  )
+  val option_picker_element: OptionPickerElement? = null,
+  @field:WireField(
+    tag = 11,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}DetailRowElement#ADAPTER"
+  )
+  val detail_row_element: DetailRowElement? = null,
+  @field:WireField(
+    tag = 12,
+    adapter = "com.squareup.wire.protos.kotlin.Form${'$'}CurrencyConversionFlagsElement#ADAPTER"
+  )
+  val currency_conversion_flags_element: CurrencyConversionFlagsElement? = null,
+  val unknownFields: ByteString = ByteString.EMPTY
+) : Message<Form, Form.Builder>(ADAPTER, unknownFields) {
+  init {
+    require(countNonNull(button_element, local_image_element, remote_image_element, money_element,
+        spacer_element, text_element, customized_card_element, address_element, text_input_element,
+        option_picker_element, detail_row_element, currency_conversion_flags_element) <= 1) {
+      "At most one of button_element, local_image_element, remote_image_element, money_element, spacer_element, text_element, customized_card_element, address_element, text_input_element, option_picker_element, detail_row_element, currency_conversion_flags_element may be non-null"
```

Comment:
hmmm, there's no wrapping though? Shouldn't we break lines?

Reply:
Nope, strings don't wrap.